hca/util/__init__.py

```diff
@@ -514,11 +537,14 @@ def _build_client_method(self, http_method, http_path, method_data):
         path_parameters = [p_name for p_name, p_data in parameters.items() if p_data["in"] == "path"]
         self.http_paths[method_name][frozenset(path_parameters)] = http_path
 
-        body_props, method_args = self._process_method_args(parameters=parameters, body_json_schema=body_json_schema)
-
         method_supports_pagination = True if str(requests.codes.partial) in method_data["responses"] else False
         highlight_streaming_support = True if str(requests.codes.found) in method_data["responses"] else False
 
+        if method_supports_pagination:
+            parameters['page'] = dict(description='Use the internal CLI paging mechanism', required=False,
+                                      name='page', default="True", type=str)
```

Comment:
Consider using this instead:

add_argument("--no-paginate", action="store_false", dest="paginate")

This will allow you to say `hca dss post-search --no-paginate ...` and the argument to consume will be `args.paginate`, a boolean.

Reply:
Much cleaner, thanks!